recipes/felupe/meta.yaml

```diff
@@ -1,5 +1,6 @@
 {% set name = "felupe" %}
 {% set version = "9.2.0" %}
+{% set python_min = "3.9" %}
```

Comment:
This one need to be set only if the package's min Python is higher than conda-forge's min.

```suggestion
```